chloggen/cmd/root.go

```diff
@@ -26,9 +26,11 @@ var (
 )
 
 var rootCmd = &cobra.Command{
-	Use:   "chloggen",
-	Short: "Updates CHANGELOG.MD to include all new changes",
-	Long:  `chloggen is a tool used to automate the generation of CHANGELOG files using individual yaml files as the source.`,
+	Use:           "chloggen",
+	Short:         "Updates CHANGELOG.MD to include all new changes",
+	Long:          `chloggen is a tool used to automate the generation of CHANGELOG files using individual yaml files as the source.`,
+	SilenceUsage:  true,
+	SilenceErrors: true,
```

Comment:
Is this an unrelated change?

Reply:
Although unrelated, but added as it was only a minor change
See my PR info:

> I have also added SilenceUsage and SilenceErrors in the root command to suppress the usage when error occurs.
> For eg:
> ```
> Error: stat ~/code/opentelemetry-go-build-tools/chloggen/.chloggen: no such file or directory
> Usage:
>   chloggen validate [flags]
> 
> Flags:
>   -h, --help   help for validate
> 
> Global Flags:
>       --chloggen-directory string   directory containing unreleased change log entries (default: .chloggen)
> 
> Error: stat ~/code/opentelemetry-go-build-tools/chloggen/.chloggen: no such file or directory
> exit status 1
> ```

Reply:
@neelayu, do you mind removing these? As minor of a change as it may be, a separate PR is appropriate.